dom.bs

```diff
@@ -6870,11 +6918,17 @@ method steps are:
   <p>If <var>attribute</var> is null, then:
 
   <ol>
-   <li><p>If <var>force</var> is not given or is true, create an <a>attribute</a> whose
+   <li><p>Set <var>attribute</var> to an <a>attribute</a> whose
```

Comment:
This covers toggleAttribute. I need to double check this because I think it could still lead to a weird state in it's current form.

Reply:
Have updated this to work how chrome appears to

Reply:
The TT-check in Chromium's `toggleAttribute` was unintentional. It came about as a side effect of how `toggleAttribute` was implemented on top of other methods. We can't come up with any XSS based on `toggleAttribute`. It'd probably be best to remove it from the PR, and that we fix our implementation on our end.

Reply:
I've reverted this now so there's no enforcement in the spec. I will do a WebKit PR to remove the enforcement from there. And the existing chromium issue that discussed some of this can be used to remove it from Chromium.